tests/tag-publish-docker-image/index.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
True, it's not explicit. Currently, tests are passing if no diff are shown. I'll improve that.